<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tasks</class>
 <widget class="QWidget" name="tasks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>571</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:13pt;
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="taskslistWidget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>140</y>
     <width>361</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>440</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt;
background : #018FFF;
color:white;
border-radius:10px;
font: 87 18pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Сохранить изменения</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>80</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt;
background : #018FFF;
color:white;
border-radius:10px;
font: 87 13pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Добавить задачу</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background : #018FFF;
color:white;
border-radius:8px;
font: 87 30pt &quot;Arial Black&quot;;
</string>
   </property>
   <property name="text">
    <string>Ежедневник</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt;
background : red;
color:white;
border-radius:10px;
font: 87 10pt &quot;Arial Black&quot;;
</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt;
background : #018FFF;
color:white;
border-radius:10px;
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt;
background : green;
color:white;
border-radius:10px;
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Найти задачу</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showCompletedButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt;
background : green;
color:white;
border-radius:10px;
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Выполненные</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showUncompletedButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>110</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt;
background : red;
color:white;
border-radius:10px;
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Не выполненные</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
